### ISSUE TITLE
Simulation with multiple .step parameters

### ISSUE THREAD

**Issue by StruchkovNS** [DongHoonPark/ltspice_pytool#24]
Dear Donghoon Park,

Thank you for creating such a useful tool!
I'm trying to get the results of simulation in "DC operating point" mode with a few varied (.STEP) parameters (example below).
.get_data('I(V1)') returns only the result of the first .STEP variation (with the start value of the second one): the upper ones in LTSpice exported .txt in the example below. Is there a way to access the rest of the data? 
For now, solved the problem by multiple simulations with different values of the second parameter. But access to all the data in one simulation would make it easier.

Best regards,
Nikolai Struchkov

![image](https://user-images.githubusercontent.com/98623012/151638224-357001d9-5ffc-497d-85bb-fe89b617c441.png)

**Comment by donghoonpark** [comment]
I totally agree with you that ltspice python tool can be improved with support of .step parameter. Until now, I knew that step parameter is only described in circuit (.asc) file, however, in such case like you, It looks like have parameter information in result file. I will develop the feature that can read parameter described in result file soon and thank you for sharing your valuable use case. 
Regards.
